Cargo.toml

```diff
@@ -36,7 +38,6 @@ json_data_content = []
 ssl = ["ssl_providerless", "rustls/ring"]
 aws-lc-rs = ["ssl_providerless", "rustls/aws-lc-rs"]
 ssl_providerless = ["home", "hyper-rustls", "rustls", "rustls-native-certs", "rustls-pemfile", "rustls-pki-types"]
-webpki = ["ssl", "dep:webpki-roots"]
```

Comment:
Fair point about these being unrecommended for production, but let's keep these for the moment in case someone is actually using them.

Reply:
Sure, I re-add it. Maybe this should be removed during the next breaking release

Reply:
Let's have a separate issue and changeset for this and communicate the change in the README.